test/models/test_vqvae.py

```diff
@@ -14,18 +14,47 @@
 from torchmultimodal.modules.layers.codebook import CodebookOutput
 
 
-@pytest.fixture(scope="class")
+@pytest.fixture(autouse=True)
 def random():
     set_rng_seed(4)
 
 
+@pytest.fixture
+def num_embeddings():
+    return 4
+
+
+@pytest.fixture
+def embedding_dim():
+    return 2
+
+
+@pytest.fixture
+def encoder():
+    enc = nn.Linear(2, 2, bias=False)
+    enc.weight = nn.Parameter(torch.tensor([[1.0, 2.0], [3.0, 4.0]]))
+    return enc
+
+
+@pytest.fixture
+def decoder():
+    dec = nn.Linear(2, 2, bias=False)
+    dec.weight = nn.Parameter(torch.tensor([[1.0, 2.0], [3.0, 4.0]]))
+    return dec
+
+
+@pytest.fixture
+def vqvae(encoder, decoder, num_embeddings, embedding_dim):
```

Comment:
nit: since this is vqvae specific, should we group it under the `TestVQVAE` class?